products/samsungmobile.md

```diff
@@ -96,8 +316,13 @@ releases:
     eol: false
     releaseDate: 2021-06-03
 
+-   releaseCycle: "Galaxy F52 5G"
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
+    eol: false # TODO
+    releaseDate: 2021-06-01
+
 -   releaseCycle: "Galaxy Tab A7 Lite"
-    support: true
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
```

Comment:
This one is upgradable to Android 12 according to https://www.gsmarena.com/samsung_galaxy_tab_a7_lite-10933.php. Should we consider it supported ?

Reply:
> I think the biggest issue is the full support section, if you agree with my comment in one of your comments, I'll change tit o state whenever applicable that a device that has received android 12 is fully supported for now, and then we change the date appropriately when we see if it receives the next major android update.

Sounds like a good rule to start. Will document the rule and fix the model if they are compatible with Android 12.